models/games.models.js

```diff
@@ -71,3 +71,22 @@ exports.fetchCommentsByReviewId = (review_id) => {
         return comments.rows
     })
 }
+exports.updateReviewById = (inc_votes, review_id) => {
+    
+    return db.query(`
+        UPDATE reviews
+        SET votes = votes + $1
+        WHERE review_id = $2
+        RETURNING*;
+    `, [inc_votes, review_id]).then((reviews) => {
+
+        if (reviews.rows.length === 0){
+            return Promise.reject({
+                status: 404,
+                msg:"Invalid review id given"
```

Comment:
The status code is good - but message should reflect this. Nothing invalid, just not found